packages/utils/src/common/sharding.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { expect } from "chai";
```

Comment:
You can keep this test in `packages/tests` similarly how other utils are done there.
https://github.com/waku-org/js-waku/blob/master/packages/tests/tests/utils.spec.ts

Reply:
@weboko I tired this but now it takes a lot longer to run the unit tests for this function. IMO it shouldn't share a test file that needs docker running in order to not fail.